scripts/validate_cclw_sheet/download.py

```diff
@@ -0,0 +1,68 @@
+"""
+Script to download the Google Sheet
+
+Requires:
+1) A Service Account to be configured
+2) The sheet in question has been "Shared" with this account's email
+3) GOOGLE_SERVICE_ACCOUNT is set in the current environment, with the 
+    contents of the json service account details (downloaded when created)
+"""
+from __future__ import print_function
+import json
+
+from os import environ
+
+# https://developers.google.com/sheets/api/quickstart/python
+# https://googleapis.dev/python/google-auth/latest/_modules/google/oauth2/service_account.html
+from google.oauth2 import service_account
+from googleapiclient.discovery import build, Resource
+from googleapiclient.errors import HttpError
+
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly"]
+
+# The ID and range of a sample spreadsheet.
+SPREADSHEET_ID = "1l0pu47JzAo_tchuRqaJh8moZM13OvEDv9DhUdydnuhQ"
+RANGE_NAME = "All new docs as of 030323 - NEW FORMAT!A:Z"
+
+
+def double_quote_quotes(col: str) -> str:
+    return col.replace('"', '""')
+
+
+def quote_string(col: str) -> str:
```

Comment:
Add as double_quote_quotes as decorator?

Reply:
we could - but its probably less transparent as to what is going on.